sonar-dotnet-shared-library/src/main/java/org/sonarsource/dotnet/shared/plugins/RoslynProfileExporter.java

```diff
@@ -286,8 +307,12 @@ private static void appendParameters(StringBuilder sb, Map<String, String> param
     return result;
   }
 
-  private List<String> allRuleKeysByRepositoryKey(String repositoryKey) {
+  private List<String> allRuleKeysByRepositoryKey(@Nullable String repositoryKey) {
```

Comment:
Why is this nullable now but not before?

Reply:
it was before as well, triggering `squid:S4449`. I just cleaned it as I was modifying the file...